packages/forms/src/fields/other/MediaInput.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { composeEventHandlers } from '@zendeskgarden/container-selection';
+import useFieldContext from '../../utils/useFieldContext';
+import { StyledTextMediaInput, StyledTextMediaFigure } from '../../styled';
+import FauxInput from './FauxInput';
+
+/**
+ * Accepts all <input> props.
```

Comment:
Needs the backticks around `<input />` otherwise styleguidist renders an actual chexkbox